app/src/components/market/common/history_chart/index.tsx

```diff
@@ -143,6 +159,9 @@ export const HistoryChart: React.FC<Props> = ({
   if (!data || status === 'Loading' || sharesDataLoader) {
     return <CustomInlineLoading message="Loading Trade History" />
   }
+  if (hasEnoughData) {
```

Comment:
hasEnoughData means it actually has data, this prop should be nonEnoughData or doesNotHaveEnoughData